doc/tutorials.rst

```diff
@@ -20,7 +20,7 @@ plugins, and devices, via simple and easy-to-follow examples.
 
 .. customgalleryitem::
     :tooltip: Use quantum machine learning to tune a beamsplitter.
-    :figure: ../examples/figures/gaussian_transformation.png
+    :figure: ../examples/figures/gaussiancircuit.png
```

Comment:
Possible to change the right side of the ket to be the standard `\rangle` instead of `>`? Looks a bit off

Reply:
on it

Reply:
@co9olguy If the new figure is ok with you, i will delete this one. I did not delete it yet in case you don't like the updates one :)

Reply:
Looks good :)